runtime/kusama/src/lib.rs

```diff
@@ -1758,6 +1762,9 @@ sp_api::impl_runtime_apis! {
 		{
 			parachains_runtime_api_impl::validation_code_hash::<Runtime>(para_id, assumption)
 		}
+
+		fn staging_get_session_disputes() -> Vec<(SessionIndex, CandidateHash, DisputeState<BlockNumber>)> {
+			staging_parachains_runtime_api_impl::get_session_disputes::<Runtime>()		}
```

Comment:
On the production runtimes (Polkadot, Kusama, Westend, ~Rococo), I wonder if we can do something like return `unimplemented!()` which will lead to an error higher up in the stack. This would ensure that we don't rely on unstable staging APIs. But not sure if there are potentially major negative effects of panicking in the runtime API. cc @bkchr